radiomanager/radiomanager/FlaskExtensions.py

```diff
@@ -0,0 +1,10 @@
+from flask_restful import Api
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+api = Api()
+
+@classmethod
```

Comment:
This doesn't make any sense: after moving this, you now have a `classmethod` outside of any class. I bet the radiomanager tests (which are mostly irrelevant now anyway) are broken.

Reply:
I don't think I use this nor know what it's for. Wasn't this to stop the loop doubling back on itself?